cspice/src/coordinates.rs

```diff
@@ -1,16 +1,28 @@
 //! Functions for converting between different types of coordinates.
-use crate::spice_unsafe;
-use crate::vector::Vector3D;
+use crate::{spice_unsafe, vector::Vector3D};
 use cspice_sys::{azlrec_c, recazl_c, reclat_c, recrad_c, SpiceBoolean, SpiceDouble};
-use derive_more::{Deref, DerefMut, From, Into};
+use derive_more::{Deref, DerefMut, Into};
 
 /// Rectangular coordinates
-#[derive(Copy, Clone, Debug, Default, PartialEq, From, Into, Deref, DerefMut)]
-pub struct Rectangular(Vector3D);
+#[derive(Copy, Clone, Debug, Default, PartialEq, Into, Deref, DerefMut)]
+pub struct Rectangular(pub [SpiceDouble; 3]);
```

Comment:
Do you think this might work better with fields?

```
pub struct Rectangular {
    pub x: SpiceDouble,
    pub y: SpiceDouble,
    pub z: SpiceDouble,
}
```

No need to change as part of this PR if you don't want to - I was just wondering what your thoughts were?

Reply:
Yeah I like this change. I think the divergence from the spice convention is made up for by the improved readability of this struct. It also would make `Rectangular` follow the convention of the other coordinate structs like `RaDec`, `AzEl`, etc.

Reply:
👍